IntelliTect.TestTools.Selenate/GoogleSearch/GoogleBrowser.cs

```diff
@@ -10,30 +10,24 @@ public class GoogleBrowser : Browser
         public GoogleBrowser(BrowserType browser) : base(browser)
         {
         }
-        
-        public IWebElement FindElement(By by)
-        {
-            return Find(by).ConfigureAwait(false).GetAwaiter().GetResult();
-        }
 
-        // Instead of doing this, I should just do a separate project demonstrating async usage on something more complex than a Google search
-        public async Task<IWebElement> FindElementAsync(By by)
+        public IWebElement FindElement(By by)
```

Comment:
@PandaMagnus have you considered overriding the By element?

Reply:
I already implemented my own IWebElement once before. It did not turn out well. :laughing: